client/sites-dashboard/components/link-in-bio-banner/link-in-bio-tile-banner.tsx

```diff
@@ -14,6 +14,10 @@ const Root = styled( Banner.Root )( {
 		position: 'absolute',
 		top: 16,
 		right: 16,
+		'html[dir=rtl] & ': {
+			right: 'unset',
+			left: 16,
```

Comment:
Same as you've done in this commit, we've been using the "logical properties" CSS properties to support RTL. I do wish that emotion did a bit more for us in terms of RTL support though. The auto RTL tooling we use for the rest of Calypso (non-Emotion code) kinda just works by magic.

TIL there are logical properties for absolute positioning too. You can use `inset-inline-end` instead of `right`. https://css-tricks.com/building-multi-directional-layouts/#aa-logical-positioning-properties